app/controllers/reviews_controller.rb

```diff
@@ -161,6 +161,11 @@ def vote_down
       current_user.vote_against(@review)
     end
 
+    if ((@review.votes_against > -9) && @review.comment != "")
```

Comment:
That was a typo from testing, it should definitely be `10>=` , fixed in the latest push